packages/jest-auto-spies/src/tests/observable-spy-utils.spec.ts

```diff
@@ -159,12 +165,15 @@ describe('createSpyFromClass - Observables', () => {
       });
 
       describe('GIVEN calledWith of nextWith is configured with the wrong params', () => {
+        let expectedMethodName: string;
```

Comment:
Usually when a variable is being used more than 3 times I tend to declare it in the nearest common `describe` to reduce a few more lines of code

Reply:
I will move it to the common describe, also in the version for jasmine. To me having a number feels a bit arbitrary, nonetheless I never know when in my tests should I extract something to a more global scope vs keeping things together where they belong.